src/vsphere_cpi/spec/unit/cloud/vsphere/soap_stub_spec.rb

```diff
@@ -8,21 +8,27 @@
   let(:retryable_adapter) { instance_double(VSphereCloud::SdkHelpers::RetryableStubAdapter) }
 
   describe '#create' do
-    it 'returns the SDK Soap Adapter' do
-      expect(VimSdk::Soap::StubAdapter).to receive(:new)
-        .with(
-          'https://some-host/sdk/vimService',
-          'vim.version.version12',
-          http_client,
-        )
-        .and_return(base_adapter)
-      expect(VSphereCloud::SdkHelpers::RetryableStubAdapter).to receive(:new)
-       .with(
-         base_adapter,
-       )
-       .and_return(retryable_adapter)
-
-      expect(soap_stub.create).to eq(retryable_adapter)
+    before(:all) do
+      @old_vc_version = $vc_version
+    end
+    after(:all) do
+      $vc_version = @old_vc_version
+    end
+    context 'when vc_version is set to 7.0' do
+      before(:each) { $vc_version = '7.0' }
+      it 'returns the SDK Soap Adapter with VIM version set to vim.version.v7_0' do
+        expect(VimSdk::Soap::StubAdapter).to receive(:new).with('https://some-host/sdk/vimService', 'vim.version.v7_0', http_client,).and_return(base_adapter)
```

Comment:
No need for a trailing comma here and on subsequent lines in the parentheses around a method call unless the arguments are split on separate lines. Though given how long this line is, we should probably split some stuff onto separate lines.